<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KSJDemoQT_TriggerMode</class>
 <widget class="QDialog" name="KSJDemoQT_TriggerMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KSJDemoQT</string>
  </property>
  <widget class="QWidget" name="PREVIEWWND_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>501</width>
     <height>561</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="DEVICENUM_Label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Device Num:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DEVICE_LIST_ComboBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="ExposureLinesSpinBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>100</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="GainSpinBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>170</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Gain:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure Time(ms):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure Lines:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ExposureTimeSpinBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range(ms):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>100</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>170</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range:</string>
   </property>
  </widget>
  <widget class="QLabel" name="EXPOSURE_TIME_RANGE_Label">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range</string>
   </property>
  </widget>
  <widget class="QLabel" name="EXPOSURE_LINES_RANGE_Label">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range</string>
   </property>
  </widget>
  <widget class="QLabel" name="GAIN_RANGE_Label">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Range</string>
   </property>
  </widget>
  <widget class="QLabel" name="EXPOSURE_TIME_REAL_Label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>140</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>REAL EXP TIME</string>
   </property>
  </widget>
  <widget class="QLabel" name="FOV_RANGE_Label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>200</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>EXP LINES RANGE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>200</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>FOV Range:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Preview</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Col Start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Row Start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Col Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Row Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MultiFrame:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetPreviewFovButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Preview Fov</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="PreviewColStartSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="PreviewColSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="PreviewRowStartSpinBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="PreviewRowSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="PreviewMultiFrameSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>370</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Capture</string>
   </property>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Col Start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Row Start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Col Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Row Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MultiFrame:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetCaptureFovButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Capture Fov</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="CaptureColStartSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="CaptureColSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="CaptureRowStartSpinBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="CaptureRowSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="CaptureMultiFrameSpinBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="PreviewButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CaptureButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QLabel" name="ERROR_INFO_Label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>530</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Error Info</string>
   </property>
  </widget>
  <widget class="QLabel" name="ELAPSE_TIME_Label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>530</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Elapse</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="FunctionTableWidget">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>0</y>
     <width>281</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="SaveCheckBox">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>450</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>230</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Flash Control:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="InvertCheckBox">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Invert</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="EnableCheckBox">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>260</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Read Timeout:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="TimeoutComboBox">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>260</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="RecoverCheckBox">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>260</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Recover</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>290</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Trigger Mode:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="TriggerModeComboBox">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>290</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>320</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Trigger Method:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="TriggerMethodComboBox">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>320</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>360</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Trigger Delay:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="DelaySpinBox">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>360</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>390</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Trigger Delay Range:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DelayRangeEdit">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>390</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="RateSpinBox">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>420</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>420</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Fixed Frame Rate(HW):</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="StatusCheckBox">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>450</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Frame Buffer Status</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>480</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Frame Buffer Status</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="EmptyButton">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>510</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Empty Frame Buffer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
